tktitler.py

```diff
@@ -72,18 +77,26 @@ def prefix(title, gfyear=None, type=PREFIXTYPE_NORMAL):
     root = _funny_substitute(root)
     age = gfyear - period
 
+    if type == PREFIXTYPE_TEX:
+        root = _escape_tex(root)
+
     def identity(n):
         return n
 
     def unicode_superscript(n):
         digits = '⁰¹²³⁴⁵⁶⁷⁸⁹'
         return ''.join(digits[int(i)] for i in str(n))
 
+    def tex_superscript(n):
+        return '$^{%s}$' % (n,)
```

Comment:
Jeg ville komme med en kommentar om at det her burde være i `\ensuremath{…}` istedet for `$…$`, så vil jeg komme med en kommentar om at det faktisk burde være `\textsuperscript{…}`, så kom jeg til at kigge i `tket.sty` og så at den bruger `$…$`. Nu er jeg i tvivl. Forskellen er ikke rigtigt et problem i praksis. `$…$`, må være godt nok.